pkg/cpu/cpu.go

```diff
@@ -141,3 +143,13 @@ func GetLoad() Load {
 		Error: "",
 	}
 }
+
+//GetCPUHardware fetches the Hardware info
+func GetCPUHardware() ([]cpuinfo.InfoStat, error) {
```

Comment:
```suggestion
func GetCPUHardware() ([]cpuinfo.InfoStat, error) {
	return cpuinfo.Info()
}
```
.. .or honestly, do we need this function?

Reply:
removed